src/main/java/ca/corefacility/bioinformatics/irida/config/services/IridaApiServicesConfig.java

```diff
@@ -176,7 +157,7 @@ public MessageSource messageSource() {
 			source.setBasenames(RESOURCE_LOCATIONS);
 		}
 
-		source.setFallbackToSystemLocale(false);
+		source.setFallbackToSystemLocale(true);
```

Comment:
What happens if the system locale is not a valid language (if this is possible)?

Reply:
To go one step further, it might be an issue if the system locale is a **valid** non_en locale as it wouldn't fall back to  `en`.  I think the right thing to do is to set this back to `false`, then rename `messages_en.properties` to `messages.properties`.  That makes it the global default (see https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/support/AbstractResourceBasedMessageSource.html#setFallbackToSystemLocale-boolean-). 

If we had full `messages_x.properties` for all (reasonable) languages the system locale would make sense, but since the other languages will likely be partial files I think this makes sense.  Thoughts?

Reply:
I think that is a really good idea.

Reply:
Done in 11b9b5d